src/entities/orderItem.ts

```diff
@@ -0,0 +1,33 @@
+import { Entity, ManyToOne, Property, Ref, Reference } from "@mikro-orm/core";
+import { Order } from "./order";
+import { Product } from "./product";
+
+@Entity()
+export class OrderItem {
+  @ManyToOne(() => Order, {primary: true, ref: true})
+  order!: Ref<Order>;
```

Comment:
M: I guess you wrongly used the order as the primary key here and this is failing any time you try to create an order with more than 2 items

Reply:
Fixed!